### PR TITLE
Add Transparent Background Mode

### LINKED ISSUE #5: after installing it my terminal loses the transparency.
I tried that. but after installing it with vim plug, my terminal lose it's transparency. These theme forced my terminal by making it a blue opaque background. whcih is not what i want..

### PR DESCRIPTION
**Pull request by niyabits**
Fixes #5 

- [x] Update Readme
- [x] Include Screenshots in the PR

### CODE REVIEW

File: lua/calvera/theme.lua [hunk 1]
```diff
@@ -108,7 +108,7 @@ theme.loadEditor = function ()
 		FoldColumn =			{ fg = calvera.blue }, -- 'foldcolumn'
 		IncSearch =				{ fg = calvera.highlight, bg = calvera.white, style = 'reverse' }, -- 'incsearch' highlighting; also used for the text replaced with ":s///c"
 		LineNr =				{ fg = calvera.line_numbers }, -- Line number for ":number" and ":#" commands, and when 'number' or 'relativenumber' option is set.
-		CursorLineNr =			{ fg = calvera.accent }, -- Like LineNr when 'cursorline' or 'relativenumber' is set for the cursor line.
+		CursorLineNr =			{ fg = calvera.white }, -- Like LineNr when 'cursorline' or 'relativenumber' is set for the cursor line.
```
Comment (SmiteshP) [inline lua/calvera/theme.lua]: instead of white, calvera.blue would look much nicer in my opinion

Comment (niyabits) [reply lua/calvera/theme.lua]: This was an unintended change tbh, I will definitely try blue and compare it with accent.

Comment (niyabits) [reply lua/calvera/theme.lua]: 🤔 I think changing `LineNr` to accent will be better too!

**Comment by niyabits** [comment]
@SmiteshP I have changed the active (curosr) line number to blue and the usual line numbers to accent :)

image

**Comment by SmiteshP** [comment]
Looks great! 👍🏽

**Pull request merged**